src/storage/v2/edge_accessor.cpp

```diff
@@ -145,6 +147,44 @@ Result<bool> EdgeAccessor::InitProperties(const std::map<storage::PropertyId, st
   return true;
 }
 
+Result<std::vector<std::tuple<PropertyId, PropertyValue, PropertyValue>>> EdgeAccessor::UpdateProperties(
```

Comment:
This method looks to me the same as for the vertex_accessor. Maybe from a code perspective it would be good to push it down to a common layer - maybe PropertyStore. If that is infeasible, then global util method is fine, as long as we remove duplication